examples/hydra/conf/config.yaml

```
@@ -0,0 +1,32 @@
+defaults:
+  - override hydra/sweeper: optuna
```

Could you remove `override`? It will be introduced by `hydra-core==1.1.0`, but it has not been released yet.
c.f., https://github.com/facebookresearch/hydra/pull/1294#discussion_r567039579

```suggestion
  - hydra/sweeper: optuna
```

It causes the following error with the current `hydra-core==1.0.6`:

```console
Key 'optuna_config' not in 'BasicSweeperConf'
        full_key: hydra.sweeper.optuna_config
        reference_type=Optional[BasicSweeperConf]
        object_type=BasicSweeperConf

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
```

Links:
- [Optuna plugin config in v1.0_branch](https://github.com/facebookresearch/hydra/blob/3b09a7e856e68451fb707c29bc1068b1086723ce/plugins/hydra_optuna_sweeper/example/conf/config.yaml#L1-L2)
- [Optuna plugin config in the master branch](https://github.com/facebookresearch/hydra/blob/51275abf21be714aa6c332d6cc91a8a791ce61de/plugins/hydra_optuna_sweeper/example/conf/config.yaml#L1-L2)

---

Thank you for your suggestion! 
I have updated this PR by 3d94b14